client/src/util/toasts/withToastProvider.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useMemo, createPortal } from 'preact/compat'
+
+import ToastContext from './context'
+import Toast from './Toast'
+
+// Create a random ID
+function generateUEID () {
```

Comment:
We should investigate returning a reference or a `Symbol`, because generating a string ID seems quite flaky.

Reply:
I'll make an issue for this